js/classes/bounding_box.js

```diff
@@ -26,6 +31,7 @@ class BoundingBox {
      * @returns true if the two bounding boxes intersect; else false.
      */
     collide(other) {
+        if (this.disabled) return false;
```

Comment:
why would you disable a bounding box?

Reply:
@NathanHinthorne did you do this so that you can drop the player through the blocks if they press a specific key? Like in some other games you can do that.